<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-4KD4H5I" ignored="0" name="test homepage" tests="5" failures="1" timestamp="04 Jan 2022 11:07:31 GMT" time="10.816" errors="0">
  <testcase name="TC01" time="0.012" classname="testCases.Validate_HomePage"/>
  <testcase name="TC02" time="0.167" classname="testCases.Validate_ContactUs"/>
  <testcase name="TC03" time="0.184" classname="testCases.Validate_ContactUs">
    <skipped/>
  </testcase> <!-- TC03 -->
  <testcase name="TC03" time="0.104" classname="testCases.Validate_ContactUs">
    <skipped/>
  </testcase> <!-- TC03 -->
  <testcase name="TC03" time="0.106" classname="testCases.Validate_ContactUs">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	null">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	null
	at testCases.Validate_ContactUs.TC03(Validate_ContactUs.java:38)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- TC03 -->
</testsuite> <!-- test homepage -->
